mmv1/products/networksecurity/FirewallEndpoint.yaml

```diff
@@ -84,6 +84,10 @@ properties:
     description: |
       Server-defined URL of this resource.
     output: true
+  - !ruby/object:Api::Type::String
+    name: 'description'
```

Comment:
would you mind adding this field in a test?

Reply:
I removed this field from this PR because API is not saving this value and causes tests to fail